build.xml

```diff
@@ -163,23 +126,6 @@
     <delete file="libraries" />
   </target>
 
-  <!-- Builds a Lightning code base from legacy Drush make files. -->
```

Comment:
Why are we getting rid of the build-legacy target? This still has the same relevance it did when we created it.